docs/generated/api.json

```diff
@@ -223,8 +223,8 @@
                 },
                 {
                   "kind": "Reference",
-                  "text": "CloudEventsContext",
-                  "canonicalReference": "@google-cloud/functions-framework!CloudEventsContext:interface"
+                  "text": "CloudEvent",
+                  "canonicalReference": "cloudevents!CloudEvent:class"

```

Comment:
This should be an `:interface`

Reply:
Looks like CI didn't catch this, thanks!

Fixing CI with https://github.com/GoogleCloudPlatform/functions-framework-nodejs/pull/401.